Client/Application/AppDelegate.swift

```diff
@@ -120,6 +120,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // Ecosia
         FeatureManagement.fetchConfiguration()
         MMP.sendSession()
+        Task.detached {
+            try? await Statistics.shared.fetchAndUpdate()
```

Comment:
Imho it's being called to often with every `appDidBecomeActive` - the values change very infrequently. So I'd advocate to put this on `applicationDidFinishLaunching` instead. Or is there is another reason to put this here?

Reply:
Thanks for the suggestion! I agree that `applicationDidFinishLaunching` is a better place, will move it.

Fetching it somewhere before it is used was introduced since there was a bug where, the first time something accessed the object, it always used the default values as the update was done asynchronously on initialization.